device_doctor/lib/src/process_helper.dart

```diff
@@ -67,3 +61,56 @@ Future<String> eval(String executable, List<String> arguments,
 
   return output.trimRight();
 }
+
+/// Splits [from] into lines and selects those that contain [pattern].
+Iterable<String> grep(Pattern pattern, {@required String from}) {
+  return from.split('\n').where((String line) {
+    return line.contains(pattern);
+  });
+}
+
+Future<bool> killAllRunningProcessesOnWindows(String processName, {ProcessManager processManager}) async {
```

Comment:
Can we move this to a different file? this seems like host related functionality rather than device or generic.

Reply:
Moved device related functionalities to `utils.dart`, and renamed this one to `host_utils.dart`.